config/jobs/kubernetes-sigs/release-notes/release-notes-presubmits.yaml

```diff
@@ -0,0 +1,47 @@
+---
+config: &config
+  always_run: true
+  decorate: true
+
+image: &image node:11
+
+command: &command
+  - /bin/bash
+  - -c
+  - >
+    npm install &&
+    npm run $JOB_NAME
+
+spec: &spec
+  <<: *config
```

Comment:
Please follow the style other yaml files in this repo and avoid using advanced yaml functionality like anchors, templating, etc.

These json objects that allow comments.

Reply:
Changed it according to the notes.